README.md

```diff
@@ -8,6 +8,10 @@ We use docker images for the software dependencies since we will not be modifyin
 
 So the steps are:
 
+#### Customize the config file
+
+If you wish to test a config file, either find or create it, then alter the ```CONFIG_URL``` to the appropriate place, should be similar to:```https://raw.githubusercontent.com/{user/repo/branch}/configs/{name_of_study}.nrel-op.json```
```

Comment:
don't you also need to tell people how to specify the `study_config` GET parameter?

Reply:
That was the only piece of code I ended up needed to change to see the join page, I'm still confused by the GET parameter I left this line as-is
`const param_study_name = new URL(connectUrl).searchParams.get("study_config");`

Reply:
Parameters are passed in to a GET HTTP request using the `?name1=value1&name2=value2&name3=value3` syntax.
(also called query parameters)
https://en.wikipedia.org/wiki/Query_string

You are right that you didn't need to change any other code.
But you did need to pass in the study config - otherwise it would have only used stage-param
(e.g. you had to use http://localhost.../?study_config=<program-name>) or similar

Reply:
Ah, ok, now I understand. I had actually been hard-coding the ENTIRE url to the config file into the CONFIG_URL (and therefore bypassing the use of the GET parameter). Now I understand what I was supposed to be doing, I'll update the readme accordingly! Thank you for explaining and pointing out that oversight!